static/app/views/organizationIntegrations/integrationDetailedView.tsx

```diff
@@ -25,7 +27,7 @@ class IntegrationDetailedView extends AbstractIntegrationDetailedView<
 > {
   getEndpoints(): ([string, string, any] | [string, string])[] {
```

Comment:
You can probably remove this returned typing since it'll be inferred now more accurately